Bluetooth: Mesh: UX issues in Sensor and Light Fixture samples (#14770)
nrfconnect/sdk-nrf: samples/bluetooth/mesh/light_ctrl/README.rst

The Sensor Setup Server model is now configured and able to receive sensor setting messages from the Sensor Client.

Occupancy Mode
--------------

> You can combine this sample with the :ref:`bluetooth_mesh_sensor_server` sample to trigger the :ref:`Occupancy On <bt_mesh_light_ctrl_srv_on_event>` event on the Light LC Server by the Occupancy Sensor.

Reviewer: ```suggestion
You can combine this sample with the :ref:`bluetooth_mesh_sensor_server` sample to trigger the Occupancy :ref:`On event <bt_mesh_light_ctrl_srv_on_event>` on the Light LC Server by the Occupancy Sensor.
```

Author: I think should be "Occupancy On" event according to the spec.

Reviewer: Hmm I agree, perhaps the linked headline should be changed? (in `doc/nrf/libraries/bluetooth_services/mesh/light_ctrl_srv.rst`)

Author: I took the Occupancy On event out of On event in the light_ctrl_srv.rst. Please have a look.

Reviewer: Looks good, much more clear now :)

To do this, first configure the Light LC Server on the **Mesh Light Fixture** node:

* Bind the model to **Application Key 1**.
* Set the subscription parameters: Create a dedicated group address.

Prepare the :ref:`bluetooth_mesh_sensor_server` sample:

* Build, run and provision the **Occupancy Sensor** node as described in the sample's documentation.

Configure the Occupancy Sensor Server that is instantiated on the Element 2 of the :ref:`bluetooth_mesh_sensor_server` sample:

* Bind the model to **Application Key 1**.
* Set the publication parameters:

  * Destination/publish address: Select the same group as the Light LC Server is subscribed to.

> When the Light LC Server is not in the **Standby** state, press ``Button 2`` on the **Occupancy Sensor** node to keep the light in the **On** state.

Reviewer 2: I think this part will be difficult to understand. I have a suggestion:
```suggestion
Evaluating occupancy inputs when light is not in **Standby** state:

* Open the **Mesh Light Fixture** node in the mobile app.
* Turn ON the light by sending Generic OnOFF Set with ON value to element 2 (<=rephrase as needed). This will bring the light LC Server out of **Standby** state.
* Now, when the Light LC Server is not in the **Standby** state, press ``Button 2`` on the **Occupancy Sensor** node to keep the light in the **On** state. 
* If the Light LC Server enters the **Standby** state, Occupancy sensor inputs will not have any effect because default value of Light LC Occupancy Mode state is 0. 

When using mobile app for iOS, you can change the value of the Light LC Occupancy Mode state to "1", and see how Occupancy Sensor inputs will turn the light ON from the **Standby** state. To do this:
...
...
```

The Sensor Setup Server model is now configured and able to receive sensor setting messages from the Sensor Client.

> Occupancy Mode

Reviewer 2: While reading this, I realized that we have not explained where does this "standby" level comes from or what to expect when light goes to standby. So, I checked above and noticed some missing info as follows:

Line 228 above:

> " to 0% over five seconds" 

Should be: 
"to `CONFIG_BT_MESH_LIGHT_CTRL_SRV_LVL_STANDBY`"

> The default value of :kconfig:option:`CONFIG_BT_MESH_LIGHT_CTRL_SRV_LVL_PROLONG` is 10000 (~15%).

Should be:
"The default value of :kconfig:option:`CONFIG_BT_MESH_LIGHT_CTRL_SRV_LVL_PROLONG` is 10000 (~15%).
The default value of :kconfig:option:`CONFIG_BT_MESH_LIGHT_CTRL_SRV_LVL_STANDBY` is 0 (0%)."

* Set the subscription parameters: Create a dedicated group address.

Prepare the :ref:`bluetooth_mesh_sensor_server` sample:

* Build, run and provision the **Occupancy Sensor** node as described in the sample's documentation.

Configure the Occupancy Sensor Server that is instantiated on the Element 2 of the :ref:`bluetooth_mesh_sensor_server` sample:

* Bind the model to **Application Key 1**.
* Set the publication parameters:

  * Destination/publish address: Select the same group as the Light LC Server is subscribed to.

Evaluating occupancy inputs when light is not in **Standby** state:

* Open the **Mesh Light Fixture** node in the mobile app.
* Open the Generic OnOff Server in the second element, then tap :guilabel:`ON` at the bottom of the Generic On Off Controls.
  This will bring the Light LC Server out of **Standby** state.
* Now, when the Light LC Server is not in the **Standby** state, press ``Button 2`` on the **Occupancy Sensor** node to keep the light in the **On** state.
* If the Light LC Server enters the **Standby** state, Occupancy sensor inputs will not have any effect because the default value of the Light LC Occupancy Mode state is ``0``.

> When using the `nRF Mesh mobile app for iOS`_, you can change the value of the Light LC Occupancy Mode state to ``1``, and see how Occupancy Sensor inputs will turn the light from the **Standby** state. To do this:

Reviewer 2: ```suggestion
When using the `nRF Mesh mobile app for iOS`_, you can change the value of the Light LC Occupancy Mode state to ``1``, and see how Occupancy Sensor inputs will turn the light ON from the **Standby** state. To do this:
```

The Sensor Setup Server model is now configured and able to receive sensor setting messages from the Sensor Client.

> Occupancy Mode
> --------------

Reviewer 3: ```suggestion
.. _bluetooth_mesh_light_lc_occupancy_mode:

Occupancy mode
--------------
```

Reviewer 3: Occupancy Mode -> Occupancy mode

Author: Fixed now!

> You can combine this sample with the :ref:`bluetooth_mesh_sensor_server` sample to trigger the :ref:`Occupancy On <bt_mesh_light_ctrl_srv_occupancy_on_event>` event on the Light LC Server by the Occupancy Sensor.

Reviewer 3: ```suggestion
You can combine this sample with the :ref:`bluetooth_mesh_sensor_server` sample to trigger the :ref:`Occupancy On <bt_mesh_light_ctrl_srv_occupancy_on_event>` event on the Light LC Server by the Occupancy sensor.
```

To do this, first configure the Light LC Server on the **Mesh Light Fixture** node:

* Bind the model to **Application Key 1**.
* Set the subscription parameters: Create a dedicated group address.

Prepare the :ref:`bluetooth_mesh_sensor_server` sample:

* Build, run and provision the **Occupancy Sensor** node as described in the sample's documentation.

Configure the Occupancy Sensor Server that is instantiated on the Element 2 of the :ref:`bluetooth_mesh_sensor_server` sample:

* Bind the model to **Application Key 1**.
* Set the publication parameters:

  * Destination/publish address: Select the same group as the Light LC Server is subscribed to.

> Evaluating occupancy inputs when light is not in **Standby** state:

Reviewer 3: ```suggestion
To evaluate occupancy inputs when light is not in **Standby** state:
```

* Open the **Mesh Light Fixture** node in the mobile app.
* Open the Generic OnOff Server in the second element, then tap :guilabel:`ON` at the bottom of the Generic On Off Controls.
  This will bring the Light LC Server out of **Standby** state.
* Now, when the Light LC Server is not in the **Standby** state, press ``Button 2`` on the **Occupancy Sensor** node to keep the light in the **On** state.
* If the Light LC Server enters the **Standby** state, Occupancy sensor inputs will not have any effect because the default value of the Light LC Occupancy Mode state is ``0``.

> When using the `nRF Mesh mobile app for iOS`_, you can change the value of the Light LC Occupancy Mode state to ``1``, and see how Occupancy Sensor inputs will turn the light ON from the **Standby** state. To do this:

Reviewer 3: ```suggestion
When using the `nRF Mesh mobile app for iOS`_, you can change the value of the Light LC Occupancy Mode state to ``1``, and see how Occupancy sensor inputs will turn the light ON from the **Standby** state.
Do this in the following way:
```
